README.md

```diff
@@ -1,12 +1,18 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/AcademySoftwareFoundation/rez/blob/main/LICENSE)<br>
 [![Release](https://shields.io/github/v/release/AcademySoftwareFoundation/rez)](https://github.com/AcademySoftwareFoundation/rez/releases)
-[![Pypy Release](https://shields.io/pypi/v/rez)](https://pypi.org/project/rez)<br>
+[![Pypi Release](https://shields.io/pypi/v/rez)](https://pypi.org/project/rez)<br>
+[![Google Group](https://img.shields.io/badge/rez--config_%28retired%29-Google%20Group-blue?style=flat-square&logo=google)](https://groups.google.com/forum/#!forum/rez-config)
+[![LFX ASWF Email List](https://img.shields.io/badge/rez--discussion-lists.aswf.io-003366?style=flat-square&logo=linuxfoundation)](https://lists.aswf.io/g/rez-discussion)
+[![Slack](https://img.shields.io/badge/Slack-ASWF_%23rez-7a6800?style=flat-square&logo=slack)](https://academysoftwarefdn.slack.com/archives/C0321B828FM)
```

Comment:
This one is though. We technically want to point people to https://slack.awsf.io which is where people should go if they are not yet part of AWS Slack's workspace.